### PR TITLE
docs: uds runner

### PR DESCRIPTION
**Pull request by UncleGedd**
Adds `docs/runner.md` for the UDS runner feature

### CODE REVIEW

File: docs/runner.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+Actions can run arbitrary bash commands including in-line scripts, and the output of a command can be placed in a
+variable using the `setVariables` key
+
+```yaml
+tasks:
+  - name: foo
+    actions:
+      - cmd: |
+          ARCH=$(uname -m)
```
Comment (zachariahmiller) [inline docs/runner.md:138]: if you indent the actual shell commands to the same indentation as the |, does the syntax issue go away?

Also maybe you can update this to be `cmd: uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/'`

Comment (UncleGedd) [reply docs/runner.md:137]: good call, I didn't see that it was jacked, fixing now

**Comment by UncleGedd** [comment]
TIL Github hates <style>

File: docs/runner.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+#### Cmd
+
+Actions can run arbitrary bash commands including in-line scripts, and the output of a command can be placed in a
+variable using the `setVariables` key
+
+```yaml
+tasks:
+  - name: foo
+    actions:
+      - cmd: uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/'
```
Comment (jeff-mccoy) [inline docs/runner.md:136]: now that #160 is merged, maybe we should do some different example so ppl aren't confused and try to use that somewhere vs the built-in env variable?

Comment (UncleGedd) [reply docs/runner.md]: Updated with `echo -n 'dHdvIHdlZWtzIG5vIHByb2JsZW0=' | base64 -d`

**Review by mikevanhemert — APPROVED** [review]

**Pull request merged**